spec/swagger/v1/schemas/recipe_ingredient_schema.rb

```diff
@@ -0,0 +1,54 @@
+RECIPE_INGREDIENT_SCHEMA = {
+  type: :object,
+  properties: {
+    ingredient_id: { type: :integer, example: 1, 'x-nullable': false },
+    ingredient_quantity: { type: :integer, example: 3, 'x-nullable': true }
+  },
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Style/TrailingCommaInHashLiteral: Avoid comma after the last item of a hash.